src/enums/documentTypes.ts

```diff
@@ -33,3 +33,5 @@ export const DOCUMENT_TYPES = {
     type: DocumetTypeEnum.COOP_RULES
   }
 }
+
+export const DRS_ID_PATTERN = /^DS\d{10}$/
```

Comment:
This isn't an enum, so shouldn't be in this folder.

Is there another folder, or file, with constants? Or maybe put this in the Document Mixin?

Reply:
Good idea. Moved to Document Mixin